pkg/network/server.go

```diff
@@ -1545,34 +1564,54 @@ func (s *Server) tryInitStateSync() {
 	if s.stateSync.IsInitialized() {
 		return
 	}
-
-	s.lock.RLock()
-	heights := make([]uint32, 0, len(s.peers))
-	for p := range s.peers {
-		if p.Handshaked() {
-			heights = append(heights, p.LastBlockIndex())
+	if !s.ServerConfig.NeoFSStateFetcherCfg.Enabled || s.syncStateFetcher.IsShutdown() {
```

Comment:
Why do we use `s.syncStateFetcher.IsShutdown()` here? Could you name the case when `!s.ServerConfig.NeoFSStateFetcherCfg.Enabled` condition is not enough for state sync initialization?